euphonic/qpoint_phonon_modes.py

```diff
@@ -525,3 +526,29 @@ def from_phonopy(cls, path='.', phonon_name='band.yaml',
             path=path, phonon_name=phonon_name, phonon_format=phonon_format,
             summary_name=summary_name)
         return cls.from_dict(data)
+
+    def get_dispersion(self) -> Spectrum1DCollection:
```

Comment:
Can this method be above `to_dict`? To keep the method ordering mentioned in the `broaden` comment

Reply:
Done